content/components/page-header/react/draft.mdx

```diff
@@ -0,0 +1,315 @@
+        Main
+      </Button>
+      <IconButton size="small" aria-label="More Options" icon={KebabHorizontalIcon} />
+    </PageHeader.ContextAreaActions>
+  </PageHeader.ContextArea>
+  <PageHeader.TitleArea>
+    <PageHeader.Title>Title</PageHeader.Title>
+  </PageHeader.TitleArea>
+</PageHeader>
+```
+
+#### With context bar and actions (only visible on mobile)
+
+```jsx live drafts
+<PageHeader>
+  <PageHeader.ContextArea>
+    <PageHeader.ContextBar>
+      <Breadcrumbs>
+        <Breadcrumbs.Item href="#">...</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">primer</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">react</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">src</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">PageHeader</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">PageHeader.tsx</Breadcrumbs.Item>
+      </Breadcrumbs>
+    </PageHeader.ContextBar>
+
+    <PageHeader.ContextAreaActions>
+      <Button size="small" leadingVisual={GitBranchIcon}>
+        Main
+      </Button>
+      <IconButton size="small" aria-label="More Options" icon={KebabHorizontalIcon} />
+    </PageHeader.ContextAreaActions>
+  </PageHeader.ContextArea>
+  <PageHeader.TitleArea>
+    <PageHeader.Title>Title</PageHeader.Title>
+  </PageHeader.TitleArea>
+</PageHeader>
+```
+
+#### With a ContextArea with all possible children (always visible)
+
+```jsx live drafts
+<PageHeader>
+  <PageHeader.ContextArea hidden={false}>
+    <PageHeader.ParentLink href="http://github.com" hidden={false}>
+      Parent Link
+    </PageHeader.ParentLink>
+
+    <PageHeader.ContextBar hidden={false}>
+      <Breadcrumbs>
+        <Breadcrumbs.Item href="#">...</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">primer</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">react</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">src</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">PageHeader</Breadcrumbs.Item>
+        <Breadcrumbs.Item href="#">PageHeader.tsx</Breadcrumbs.Item>
+      </Breadcrumbs>
+    </PageHeader.ContextBar>
+
+    <PageHeader.ContextAreaActions hidden={false}>
+      <Button size="small" leadingVisual={GitBranchIcon}>
+        Main
+      </Button>
+      <IconButton size="small" aria-label="More Options" icon={KebabHorizontalIcon} />
+    </PageHeader.ContextAreaActions>
+  </PageHeader.ContextArea>
+  <PageHeader.TitleArea>
+    <PageHeader.Title>Title</PageHeader.Title>
+  </PageHeader.TitleArea>
+</PageHeader>
+```
+
+### With actions that have responsive content
+
+```jsx live drafts
+<PageHeader>
+  <PageHeader.TitleArea>
+    <PageHeader.Title as="h2">Webhooks</PageHeader.Title>
+    <PageHeader.Actions>
+      <Hidden when={['narrow']}>
+        <Button variant="primary">New webhook</Button>
+      </Hidden>
+      <Hidden when={['regular', 'wide']}>
+        <Button variant="primary">New</Button>
+      </Hidden>
+    </PageHeader.Actions>
+  </PageHeader.TitleArea>
+</PageHeader>
+```
```

Comment:
Am I correct in assuming that the prop tables are automatically rendered because this is using the default component layout component?

Reply:
Yes that's correct 👍